tests/ci_build/Dockerfile.build_cuda8_cudnn7

```diff
@@ -0,0 +1,26 @@
+FROM nvidia/cuda:8.0-cudnn7-devel
+# cuda8.0 has to be used because this is the first ubuntu16.04 container
+# which is required due to OpenBLAS being incompatible with ubuntu14.04
+# the reason we used a gpu base container because we are going to test MKLDNN
+# operator implementation against GPU implementation
```

Comment:
Please remove this file, we're not using that directory for dockerfiles anymore

Reply:
@marcoabreu It seems the required version of CUDNN is not installed properly so that tests failed. Do we still need to put this docker file under `ci/docker/` to get the required cudnn from the NV's docker hub?

Reply:
Yes. But how about we just upgrade to cudnn7? Could it cause any issues?